google/cloud/bigquery/datatransfer/v1/bigquerydatatransfer_gapic.yaml

```diff
@@ -27,20 +27,37 @@ license_header:
 collection_oneofs:
 - oneof_name: parent_oneof
   collection_names:
-  - location
   - project
+  - location
```

Comment:
Does this reordering have an important effect?

Reply:
It does - the sample code generation picks the first one in the list for the example type.

Reply:
Yes, but why do we want project instead of location?

Reply:
It's a shorter resource path with fewer required inputs, so it seems like it's the most "basic."